_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,108 @@
+---
+layout:         pattern
+title:          Alerts
+date:           2015-01-28 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- alerts
+
+slug:           alerts
+
+url_styles:     "patterns/_alerts.scss"
+
+description:    Examples of alerts in a semi-real world context.
+
+info:           Alerts provide information about...TBD
+---
+
+<h3 class="hd-6 example-set-hd">Error Alert with Title and Button</h3>
+<div class="example-set">
+    <div class="alert alert-error" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-error">
+                We are unable to load this chart.
+            </h3>
+        </div>
+
+        <button type="button" class="alert-action btn-neutral btn">Read More</button>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Success Alert with Title and Message</h3>
+<div class="example-set">
+    <div class="alert alert-success" role="alert" aria-labelledby="alert-title-success" tabindex="-1">
+        <span class="icon alert-icon icon-check" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-success">
+                Success
+            </h3>
+
+            <div class="alert-copy">
+                Your changes have been saved.
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Warning Alert with Title, Message, and Button</h3>
+<div class="example-set">
+    <div class="alert alert-warning" role="alert" aria-labelledby="alert-title-warning" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-warning">
+                Waiting for Peer Responses
+            </h3>
+
+            <p class="alert-copy">
+                All submitted peer responses have been assessed.  Check back later to see if
+                more students have submitted responses.  You'll recieve your grade after you've
+                completed all the steps for this problem and your peers have assessed your
+                responses.
+            </p>
+        </div>
+
+        <button type="button" class="alert-action btn-neutral btn">Reload</button>
+    </div>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Warning Alert with Message Only</h3>
+<div class="example-set">
+    <div class="alert alert-warning" role="alert" aria-labelledby="alert-warning-message" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
```

Comment:
@clrux, are you recommending that a title be included with the `sr` class and that it not be visible?